app/constants/view.ts

```diff
@@ -5,6 +5,9 @@ import {Platform} from 'react-native';
 
 export const BOTTOM_TAB_HEIGHT = 52;
 export const BOTTOM_TAB_ICON_SIZE = 31.2;
+export const BOTTOM_TAB_PROFILE_PHOTO_SIZE = 22;
```

Comment:
@matthewbirtch for the tab bar (only), the icon is slightly smaller (originally 28, but now 22).

And the status icon is now 12 instead of 14.

![smaller-profile-photo](https://github.com/user-attachments/assets/38b9f148-6cd7-4085-8961-da9df0acfbdb)

Reply:
Ignore the greyed background, that was before I reverted the original changes.

Reply:
The line actually looks slanted here.  So ignore this.